src/main/java/build/buildfarm/instance/shard/RedisShardSubscriber.java

```diff
@@ -62,16 +62,19 @@ void complete() {
 
   private final ListMultimap<String, TimedWatchFuture> watchers;
   private final Map<String, ShardWorker> workers;
+  private final int workerType;
```

Comment:
Rename this to workerChangeTypeMask given the behavior.

Reply:
Done. workerChangeTypeMask is better.